manifests/repos/apt.pp

```diff
@@ -10,7 +10,11 @@
   ::apt::source { $name:
     repos    => $repo,
     location => $location,
-    key      => $key,
+    key      => {
+      id     => $key,
+      source => 'http://deb.theforeman.org/foreman.asc',
```

Comment:
Could you make the key also a parameter? Also wondering if we can fetch it over https so there is at least integrity.